perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -1573,7 +1573,7 @@ public RichGroup call(ApiCaller ac, Deserializer parms) throws PerunException {
 
 			return ac.getGroupsManager().getRichGroupByIdWithAttributesByNames(ac.getSession(),
 					parms.readInt("groupId"),
-					parms.readList("attrNames", String.class));
+					null);
```

Comment:
Why is this change in this PR?

Reply:
I added this by accident. I have removed it.